docs/syntaxes.md

```diff
@@ -0,0 +1,64 @@
+# Syntaxes
+
+Melange supports two syntaxes: OCaml syntax (the default) and Reason syntax.
```

Comment:
I would add a paragraph on what we mean by syntaxes, before we say melange supports both

Reply:
I would mention that the syntax for the melange website can be toggled.

Reply:
I explain how to toggle syntaxes in https://github.com/melange-re/melange-re.github.io/pull/200/commits/5bacbf32a9df043bb0eb2a2fc561b9c20a0f0eba

> I would add a paragraph on what we mean by syntaxes, before we say melange supports both

I added an example snippet before explaining the difference in https://github.com/melange-re/melange-re.github.io/pull/200/commits/5bacbf32a9df043bb0eb2a2fc561b9c20a0f0eba. Is that good enough?